code/game/mecha/mecha_topic.dm

```diff
@@ -71,19 +71,24 @@
 /obj/mecha/proc/get_stats_part()
 	var/integrity = obj_integrity/max_integrity*100
 	var/cell_charge = get_charge()
-	var/datum/gas_mixture/int_tank_air = internal_tank.return_air()
-	var/tank_pressure = internal_tank ? round(int_tank_air.return_pressure(),0.01) : "None"
-	var/tank_temperature = internal_tank ? int_tank_air.temperature : "Unknown"
-	var/cabin_pressure = round(return_pressure(),0.01)
+	var/datum/gas_mixture/int_tank_air = 0
+	var/tank_pressure = 0
+	var/tank_temperature = 0
+	var/cabin_pressure = 0
+	if (internal_tank)
+		int_tank_air = internal_tank.return_air()
+		tank_pressure = internal_tank ? round(int_tank_air.return_pressure(),0.01) : "None"
+		tank_temperature = internal_tank ? int_tank_air.temperature : "Unknown"
+		cabin_pressure = round(return_pressure(),0.01)
 	. = {"[report_internal_damage()]
 						[integrity<30?"<span class='userdanger'>DAMAGE LEVEL CRITICAL</span><br>":null]
 						<b>Integrity: </b> [integrity]%<br>
 						<b>Powercell charge: </b>[isnull(cell_charge)?"No powercell installed":"[cell.percent()]%"]<br>
-						<b>Air source: </b>[use_internal_tank?"Internal Airtank":"Environment"]<br>
-						<b>Airtank pressure: </b>[tank_pressure]kPa<br>
-						<b>Airtank temperature: </b>[tank_temperature]&deg;K|[tank_temperature - T0C]&deg;C<br>
-						<b>Cabin pressure: </b>[cabin_pressure>WARNING_HIGH_PRESSURE ? "<span class='danger'>[cabin_pressure]</span>": cabin_pressure]kPa<br>
-						<b>Cabin temperature: </b> [return_temperature()]&deg;K|[return_temperature() - T0C]&deg;C<br>
+						<b>Air source: </b>[internal_tank?"[use_internal_tank?"Internal Airtank":"Environment"]":"Enviroment"]<br>
```

Comment:
typo, also you could probably display air pressure/temp/etc below instead of n/a

Reply:
Do you mean the outside environment pressure/temp? That's a neat idea, but if I do that, it'll probably be in a different PR. Thanks for catching the typo.